cMake/FreeCAD_Helpers/InitializeFreeCADBuildOptions.cmake

```diff
@@ -64,6 +64,7 @@ macro(InitializeFreeCADBuildOptions)
     else(MSVC)
         option(FREECAD_LIBPACK_USE "Use the LibPack to Build FreeCAD (only Win32 so far)." OFF)
         set(FREECAD_LIBPACK_DIR ""  CACHE PATH  "Directory of the FreeCAD LibPack")
+        option(FREECAD_USE_PCH "Activate precompiled headers where it's used." OFF)
```

Comment:
This doesn't make too much sense as currently PCH is only used for MSVC.

Reply:
yes, but if you make a code change to the headers and you don't have MSVC this is the only way to test if compilation succeeds - obviously its default OFF

Reply:
Not really. With CMake you can add any switch you want.
When running the command line version you can use `-DFREECAD_USE_PCH=OFF` and with the GUI version you can use the _Add entry_ button.

Reply:
I tried that. it didn't work, I added -DFREECAD_USE_PCH:BOOL=True to cmake - but it complained in a warning that that switch doesn't exist, and it still didn't add -D_PreComp_ to the Makefiles - it only did that once I added that line

Reply:
might be dependant on the cmake version

Reply:
i take it back, it does work, must have had a typo. im removing that line then